src/libraries/System.Runtime/tests/System/Reflection/ReflectionCacheTests.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    [SkipOnMono("Mono doesn't have a reflection cache")]
+    public class ReflectionCacheTests
+    {
+        [Fact]
+        public void GetMethod_MultipleCalls_SameObjects()
```

Comment:
nit: this test should pass on Mono.

Reply:
Interesting.  I was under the impression mono didn't have a reflection cache.  Thanks for correcting me.